docker/fuzzbench/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM python:3.7
```

Comment:
Which ubuntu distro does this use ? or does it not matter ?

Reply:
It uses debian, but it doesn't matter: this image is for running fuzzbench python code (see previous comment), so we just need a python runtime. This is the official python docker image (https://hub.docker.com/_/python). Let me know if you have any concerns.

Reply:
Should be ok then!